integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py

```diff
@@ -262,7 +263,8 @@ def create_span(self, context: SpanContext) -> LangfuseSpan:
 
         tracing_ctx = tracing_context_var.get({})
         if not context.parent_span:
-            if context.operation_name not in [_PIPELINE_RUN_KEY, _ASYNC_PIPELINE_RUN_KEY]:
+            allowed_span_roots = [_PIPELINE_RUN_KEY, _ASYNC_PIPELINE_RUN_KEY, _AGENT_RUN_KEY]
+            if context.operation_name not in allowed_span_roots:
```

Comment:
hey @vblagoje I think we agreed to just remove this check since it's not that useful and is often wrong. So think I we should just remove the check instead of adding Agent. This way when we add any new spans in the future this won't trigger.

Reply:
Ok deal, I'll make an update to remove it 👍